packages/cli/test/functional/testUtil/compare.js

```diff
@@ -17,7 +17,6 @@ const TEST_BLACKLIST = ignore().add([
 ]);
 
 // Files that possibly have null characters but are not binary files
```

Comment:
Let's remove this comment as well.

Reply:
Sorry I missed out the comment! I have removed it in the latest commit